src/cmake/thirdparty/SetupGeosxThirdParty.cmake

```diff
@@ -104,21 +104,41 @@ else()
     set(RAJA_DIR ${GEOSX_TPL_DIR}/raja)
 endif()
 
-include(${CMAKE_SOURCE_DIR}/cmake/thirdparty/FindRAJA.cmake)
-if (NOT RAJA_FOUND)
-    message(FATAL_ERROR "RAJA not found in ${RAJA_DIR}. Maybe you need to build it")
-endif()    
-blt_register_library( NAME raja
-                      INCLUDES ${RAJA_INCLUDE_DIRS}
-                      LIBRARIES ${RAJA_LIBRARY}
-                      TREAT_INCLUDES_AS_SYSTEM ON )
+find_package(RAJA REQUIRED PATHS ${RAJA_DIR})
+
+get_target_property(RAJA_INCLUDE_DIRS RAJA INTERFACE_INCLUDE_DIRECTORIES)
+set_target_properties(RAJA
```

Comment:
@bmhan12 do you know why this is necessary? When building just LvArray by itself the `find_package` works fine. Even when building the LvArray unit tests within geosx it works. But for some reason when building geosx RAJA and CHAI are not treated as system includes.